### ISSUE TITLE
Broken with new versions of Node

### ISSUE THREAD

**Issue by firelizzard18** [Orbs/jspm-git#56]
See also jspm/github#117.

A recent change in Node (version 8?) means that child_process.execFile will barf if options.timeout is not an integer. Thus, `jspm install` fails on these versions of Node.

```
Unhandled rejection TypeError: "timeout" must be an unsigned integer
      at validateTimeout (child_process.js:640:11)
      at exports.execFile (child_process.js:201:3)
      at C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\exec-git.js:74:9
      at Promise._execute (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\debuggability.js:300:9)
      at Promise._resolveFromExecutor (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\promise.js:483:18)
      at new Promise (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\promise.js:79:10)
      at C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\exec-git.js:73:14
      at tryCatcher (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\util.js:16:23)
      at Promise._settlePromiseFromHandler (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\promise.js:512:31)
      at Promise._settlePromise (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\promise.js:569:18)
      at Promise._settlePromiseCtx (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\promise.js:606:10)
      at Async._drainQueue (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\async.js:138:12)
      at Async._drainQueues (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\async.js:143:10)
      at Immediate.Async.drainQueues (C:\Build\trace\Ouranos.Trace.Web.Applications\node_modules\jspm-git\node_modules\bluebird\js\release\async.js:17:14)
      at runCallback (timers.js:800:20)
      at tryOnImmediate (timers.js:762:5)
      at processImmediate [as _immediateCallback] (timers.js:733:5)
```

My hack is to patch `exec-git.js` (see the lines with `// HACK`):

```javascript
if (process.platform === 'win32') {
  var gitPool = new Pool(Math.min(os.cpus().length, 2));
  module.exports = function(command, execOpt, callback) {
    execOpt = Object.assign(execOpt || {}, { timeout: 0 }); // HACK
    return gitPool.execute(function() {
      return new Promise(function(resolve){
        execFile('git', command, execOpt, function(err, stdout, stderr){
          callback(err, stdout, stderr);
          resolve();
        });
      });
    });
  };
} else {
  module.exports = function(command, execOpt, callback) {
    execOpt = Object.assign(execOpt || {}, { timeout: 0 }); // HACK
    execFile('git', command, execOpt, callback);
  };
}
```

Please fix this ASAP.

**Comment by subesokun** [comment]
Thanks for reporting the issue. There is already a PR #53 ongoing so I'll close this issue along with it once the PR has been merged.

**Comment by firelizzard18** [comment]
Awesome! That's one thing I can remove from my frankensteinian pile of gulp/regex patches.

**Comment by subesokun** [comment]
Just released v0.6.1, please give it a try.
